src/mod.ts

```diff
@@ -56,6 +56,20 @@ export type ChatMembersOptions = {
    * another chat where the bot is present.
    */
   getKey: (chatId: string | number, userId: number) => string;
+  /**
+   * Configuration options for caching chat members
+   * @default { enabled: false, ttl: 60 }
+   */
+  caching?: {
+    /**
+     * Whether or not to enable caching of chat members
+     */
+    enabled: boolean;
+    /**
+     * Time in seconds to keep chat members in cache
+     */
+    ttl?: number;
```

Comment:
Perhaps give this a better name, such as `secondsToLive`

Reply:
That makes sense.